index.html

```diff
@@ -0,0 +1,178 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Untitled Document</title>
+  <link rel="stylesheet" href="styles/styles.css">
+</head>
+
+<body>
+
+  <div class="container">
+
+    <div id="row-01" class="full-width">
+      <div class="heat-100">
+        <p>Fusce in urna sem. Vivamus vehicula dignissim augue et scelerisque.</p>
+      </div>
+    </div>
+
+    <div id="row-02" class="two-columns">
+      <div class="heat-75">
```

Comment:
Try to include image, title and article body sections withing each article.

Maybe wrap the article in `article` tag